client/X11/xf_cliprdr.c

```diff
@@ -3072,17 +3072,15 @@ static DWORD WINAPI xf_cliprdr_fuse_thread(LPVOID arg)
 		fuse_session_destroy(clipboard->fuse_sess);
 	}
 #else
-	struct fuse_chan* ch;
-	int err;
-
-	if ((ch = fuse_mount(clipboard->delegate->basePath, &args)) != NULL)
+	struct fuse_chan* ch = fuse_mount(clipboard->delegate->basePath, &args);
+	if (ch != NULL)
 	{
 		clipboard->fuse_sess = fuse_lowlevel_new(&args, &xf_cliprdr_fuse_oper,
 		                                         sizeof(xf_cliprdr_fuse_oper), (void*)clipboard);
 		if (clipboard->fuse_sess != NULL)
 		{
 			fuse_session_add_chan(clipboard->fuse_sess, ch);
-			err = fuse_session_loop(clipboard->fuse_sess);
+			const int err = fuse_session_loop(clipboard->fuse_sess);
```

Comment:
I honestly don't see a reason to make this constant here. I mean, we won't change the value here afterwards, sure, but this here is a bit unusual, we normally never do that for errors.  
In pure theory, `fuse_session_loop` could always fail, but with different error value on each run. Having this a constant here would be a bit misleading.

Reply:
just my preference (add `const` by default) to avoid overriding variables by accident. but yes, we currently do not check the return, fixing that.